rtengine/iplocallab.cc

```diff
@@ -7523,15 +7523,15 @@ BENCHFUN
         int W_L = wdspot->level_W(0);//provisory W_L H_L
         int H_L = wdspot->level_H(0);
 
-        float *betalev[12];
+        float *betalev[33];//3*10 levels + 3 dir
 
-        float *betalevbuffer = new float[12 * H_L * W_L]; //12
+        float *betalevbuffer = new float[33 * H_L * W_L]; //12
```

Comment:
@Desmis, Jacques, when your changes are completed, I will have a look whether this can be optimized, because when working with a 100 MP file, this allocation can ba more than 3 GB